### ISSUE TITLE
Enable mode filter in transit routing

### ISSUE THREAD

**Issue by kaligrafy** [chairemobilite/transition#404]
We should be able to enable/disable modes (train, bus, subway, tram, etc.) whenever we calculate transit routing (routing, od routing, batch routing and accessibility maps). trRouting can accept exceptModesIdx and onlyModesIdx parameters.
